src/constants/fonts.ts

```diff
@@ -0,0 +1,60 @@
+const fontFamily = {
+  NunitoSansExtraLight: 'NunitoSans_10pt-ExtraLight',
+  NunitoSansExtraLightItalic: 'NunitoSans_10pt-ExtraLightItalic',
+  NunitoSansLight: 'NunitoSans_10pt-Light',
+  NunitoSansLightItalic: 'NunitoSans_10pt-LightItalic',
+  NunitoSansRegular: 'NunitoSans_10pt-Regular',
+  NunitoSansRegularItalic: 'NunitoSans_10pt-RegularItalic',
+  NunitoSansMedium: 'NunitoSans_10pt-Medium',
```

Comment:
why using two font families?

Reply:
One for special heading text and one for text throughout the app